index.html

```diff
@@ -543,6 +543,12 @@
                         "maxTileRow": 9375,
                         "minTileCol": 15914,
                         "maxTileCol": 20507
+                    },
```

Comment:
Add a mechanism in the API, if  a new `tileMatrixSetLimits` is needed then it's added automatically. 
And remove this :
```js
  "20": {}
```